tests/dummy/app/controllers/application.js

```diff
@@ -138,6 +150,11 @@ export default class ApplicationController extends Controller {
   countDownAction() {
     console.log('countDownAction');
   }
+
+  @action
+  onSelectionNavTab() {
+    console.log('onSelectionNavTab');
```

Comment:
You should test that the tab selection works here, the navtab component should return the selected tab, and you can set it to selected=true from here. (and deselect the previously selected one)